### ISSUE TITLE
Regarding the problem that the results of the indicators in the pre-trained model log are much higher than the paper indicators in arxiv

### ISSUE THREAD

**Issue by xiangchiqingyu** [wgcban/ChangeFormer#31]
Hello, may I ask you, the index displayed in the log of your training model is much higher than the index in the arxiv paper. Have you updated it in the officially published paper? Wondering what your official metrics are? Looking forward to your reply

**Comment by wgcban** [comment]
@xiangchiqingyu Thanks for your question. I am wondering which matrix you are talking about. In the paper, we report performance metrics w.r.t. the **change-class** and not the average metrics of **change** and **no-change** classes. Since there is a high-class imbalance, the average metrics are usually comparably higher than the metric w.r.t. **change-class**. Pls make sure that you compare the performance metrics of **change-class** (class-1 in the log file) in the log file with the values reported in the paper.

**Comment by xiangchiqingyu** [comment]
Thank you very much for your reply
